### ISSUE TITLE
Add option to keep current grid size when changing dimensions by default

### ISSUE THREAD

**Issue by NeutralGoodGamer** [foundryvtt/foundryvtt#10294]
### User Experience

i already posted a "bug" report but ive been told that this is actually working as intended (and a common request) so im reposting as a feature request

pretty much whats in the title, whenever I resize my map it auto-scales the grid size and the other dimension (heightxwidth) by default, this has really messed up my work flow and I'd really appreciate a setting that sets the "link" or "auto-resize" or w/e its called to off by default. 

Basically i want my grid square size to ALWAYS be 100x100 and when i change a 2000x2000 map to 2100x2000 i want it to do that and not auto change to 2100x2100. Just as an option in settings
